.github/workflows/build_and_test.yml

```diff
@@ -260,6 +260,16 @@ jobs:
             cp "$SRC" "$DST"
           done
 
+          cdylibs=$(
+            ${cov_prefix} cargo metadata $CARGO_FEATURES --format-version=1 --no-deps |
+            jq -r '.packages[].targets[] | select(.kind | index("cdylib")) | .name'
+          )
+          for lib in $cdylibs; do
+            SRC=target/$BUILD_TYPE/lib$lib.so
+            DST=/tmp/neon/bin/lib$lib.so
+            cp "$SRC" "$DST"
+          done
+
```

Comment:
Needed to transport the libstatvfs_ldpreload.so into the next CI stage.

But, I guess this will be irrelevant as we'll switch to the proposed `mock_fstatfs` above.